ten-simple-rules-dockerfiles.tex

```diff
@@ -290,38 +290,37 @@ \section*{Introduction}\label{introduction}}
-While ``papers'' based on the traditional journal article format can
-share high level details of research, computational research is
-oftentimes way too complicated to be disseminated alongside in this
+is paramount to increasing the reproducibility of computational
+research. While ``papers'' based on the traditional journal article
+format can share high level details of research, computational research
+is oftentimes way too complicated to be disseminated alongside in this
 format {[}1{]}. In that the actual contribution to knowledge includes
 the full computing environment that produced a result {[}2{]},
 containerization is needed.
 
 Containerization plays an important role in providing instructions for
 packaging the building blocks of computer-based research (i.e.~code,
-data, documentation, and the computing environment). While a container
-cannot package all of these blocks, it represents a human- \emph{and}
-machine-readable recipe to capture the computing environment and
+data, documentation, and the computing environment). Containers are
+built from plain text files which represent a human- \emph{and}
+machine-readable recipe for creating the computing environment and
 interact with data. By providing this recipe, authors of scientific
 articles greatly improve their work's level of documentation,
-transparency, and reusability. Such practice is one important part of
-common practices for scientific computing {[}3,4{]}, with the result
-that it is much more likely both the author and others are able to
-reproduce and extend an analysis workflow. The containers built from
-these recipes are portable encapsulated snapshots of a specific
-computing environment. Such containers have been demonstrated for
-capturing scientific notebooks {[}5{]} and reproducible workflows
-{[}6{]}.
+transparency, and reusability. This is an important and common practice
+in scientific computing {[}3,4{]}, with the result that it is much more
+likely that both the author and others are able to reproduce and extend
+an analysis workflow. The containers built from these recipes are
+portable encapsulated snapshots of a specific computing environment.
+Such containers have been demonstrated for capturing scientific
+notebooks {[}5{]} and reproducible workflows {[}6{]}.
 
 While there are several tutorials for using containers for reproducible
 research {[}7--11{]}, there is no detailed \emph{manual for how to write
 the actual instructions to create the containers for computational
-research} besides generic best practices {[}12,13{]}. Guening
-et~al.~{[}14{]} give very helpful recommendations for packaging an
-individual reusable software in a container. This article introduces
-these instructions for the popular \texttt{Dockerfile} format with a
-focus on containers encapsulating data science workflows.
+research} besides generic best practices {[}12,13{]}. Guening et~al.
+{[}14{]} give very helpful recommendations for packaging an individual
```

Comment:
This reads funny - "individual reusable software" - maybe we can just say "for packaging reusable software" ?

Reply:
I didn't write this bit but I agree and have rephrased it.